medic/cht-core
# chore(#8459): update nginx image to latest 1.x

# Description

bump nginx image version to latest `1.x`

medic/cht-core#8459

### testing

* checkout this branch
* `cd cht-core/nginx` 
* build new image for nginx: `docker build -t 8459-nginx-bump .`
* start a new docker helper instance `8459_nginx_bump`
* change compose file to have nginx service use new image  `~/.medic/cht-docker/8459_nginx_bump-dir/compose/cht-core.yml` **_OR_** download the core-compose file from this PR - you pick!
* restart cht instance and check all is well poking around on a browser and on mobile device running CHT Android 1.x
* use `curl` to validate `nginx` is at `1.25.1`:
   ```
   curl -s -o /dev/null -v https://192-168-1-132.local-ip.medicmobile.org:10450  2>&1 | grep -i Server:
   < Server: nginx/1.25.1
   ```


# Code review checklist


# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8459-bump-nginx/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8459-bump-nginx/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8459-bump-nginx/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.
